ci/infra/testrunner/tests/test_istio_deployment.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import pytest
+import requests
+import time
+
+
+GATEWAY_HTTPBIN = ("""
+---
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: httpbin-gateway
+  namespace: default
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - hosts:
+    - '*'
+    port:
+      name: http
+      number: 80
+      protocol: HTTP
+EOF
+""")
+
+VIRTUALSERVICE_HTTPBIN = ("""
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: httpbin
+spec:
+  hosts:
+  - '*'
+  gateways:
+  - httpbin-gateway
+  http:
+  - match:
+    - uri:
+        prefix: /status
+    - uri:
+        prefix: /delay
+    route:
+    - destination:
+        port:
+          number: 8000
+        host: httpbin
+EOF
+""")
+
+
+@pytest.mark.pr
```

Comment:
Is this just for forcing the test to run in the PR pipeline or do you plan to leave it as a standard PR test? I would suggest not and run it as an `e2e` test.  PR tests should be the minimal signal to ensure the PR is not breaking the possibility to deploy a cluster, any other feature should be tested as an e2e test.

Reply:
ok